docs/kernel-policy.md

```diff
@@ -19,9 +19,9 @@ release only if compatibility changes are required.
 
 | Host kernel | Guest kernel v4.14 | Guest kernel v5.10 | Min. end of support |
 | ----------: | :----------------: | :----------------: | ------------------: |
-|       v4.14 |          Y         |          Y         |          2021-01-22 |
-|       v5.10 |          Y         |          Y         |          2024-01-31 |
-|        v6.1 |          Y         |          Y         |          2025-10-12 |
+|       v4.14 |         Y          |         Y          |          2021-01-22 |
```

Comment:
commit `fix(docs) Fix kernel supported table`: "pass the test" - what test do you mean?

Reply:
This change of space somehow was required to let test_markdown_style.py pass. 
Questionable why it is needed but please see https://buildkite.com/firecracker/firecracker-pr/builds/8828#018d6cee-58c7-4901-864e-1b50e949bd15 if you need more details. 
I just want to add that i run mdformat locally  on all the files before pushing the PR but the  outcome the table format was completely messed up so had to adjust manually.

Reply:
yeah, that's fair. It just would be better to be specific about the test in the commit message for future reference.

Reply:
I added the specific test in the commit message, thanks